hw/ip/aon_timer/data/aon_timer_testplan.hjson

```diff
@@ -12,22 +12,60 @@
     {
       name: smoke
       desc: '''
-            Smoke test accessing a major datapath within the aon_timer.
+            Smoke test initializes and starts AON Timer according to
+            the programmers guide.
 
             **Stimulus**:
-            - TBD
+            - Register writes to Watchdog/Wakeup COUNT, CTRL and THOLD registers
 
             **Checks**:
-            - TBD
+            - If we are changing WDOG_BARK_THOLD to be lower than the current WDOG_COUNT
+            watchdog timer should raise the interrupt `wdog_timer_bark` .
```

Comment:
maybe I did not comment the right place, still see some extra space before the `.`
Same for line 25.